### PR TITLE
Avoid warnings

### PR DESCRIPTION
**Pull request by bramtayl**
These changes avoid gcc and clang compilation warnings (haven't tested msvc). I separated it into several smaller commits with similar changes.

### CODE REVIEW

File: InOut/rtalsa.c [hunk 1]
```diff
@@ -1856,7 +1856,7 @@ int listAlsaSeq(CSOUND *csound, CS_MIDIDEVICE *list, int isOutput) {
       /* reset query info */
       snd_seq_port_info_set_client(pinfo, snd_seq_client_info_get_client(cinfo));
       snd_seq_port_info_set_port(pinfo, -1);
-      count = 0;
+      // count = 0;
```
Comment (stekyne) [inline InOut/rtalsa.c]: Is this not read at all? Then maybe delete. Shouldn't have commented code

File: Opcodes/OpcodeBase.hpp [hunk 2]
```diff
@@ -374,7 +379,7 @@ class OpcodeNoteoffBase
 };
 
 #if (__cplusplus >= 201103L) && !(defined(__wasi__))
-#pragma message("Defining heap_object_manager_t.")
+// #pragma message("Defining heap_object_manager_t.")
```
Comment (stekyne) [inline Opcodes/OpcodeBase.hpp]: Whats the warning here?

File: Opcodes/bformdec2.cpp
Comment (stekyne) [inline Opcodes/bformdec2.cpp:1]: Are you sure the default values make sense for the params?

File: tests/c/csound_debugger_test.cpp [hunk 3]
```diff
@@ -174,7 +180,10 @@ TEST_F (DebuggerTests, testBreakpointInstrument)
 int count = 0;
 static void brkpt_cb5(CSOUND *csound, debug_bkpt_info_t *bkpt_info, void *userdata)
 {
-    debug_opcode_t *debug_opcode = bkpt_info->currentOpcode;
+    (void)(csound);
+    (void)(bkpt_info);
+    (void)(userdata);
+    // debug_opcode_t *debug_opcode = bkpt_info->currentOpcode;
```
Comment (stekyne) [inline tests/c/csound_debugger_test.cpp]: Whats the warning here?

**Review by stekyne — APPROVED** [review]

File: InOut/rtalsa.c (hunk 1 above)
Comment (bramtayl) [reply InOut/rtalsa.c]: Ok, deleted instead of commented unused assignments

File: tests/c/csound_debugger_test.cpp (hunk 3 above)
Comment (bramtayl) [reply tests/c/csound_debugger_test.cpp]: debug_opcode was unused. Maybe it's needed for breakpoint or something though?

File: Opcodes/bformdec2.cpp
Comment (bramtayl) [reply Opcodes/bformdec2.cpp:1]: Nope. But they were undefined before, so accessing them would have caused bugs/undefined behavior anyways

File: Opcodes/OpcodeBase.hpp (hunk 2 above)
Comment (bramtayl) [reply Opcodes/OpcodeBase.hpp]: Oops, I thought I had seen somewhere that pragma messages count as warnings, but I guess I was wrong, so I reverted.

**Comment by stekyne** [comment]
@kunstmusik what do you think?

**Comment by bramtayl** [comment]
Bump

**Pull request closed without merging**